sqlglot/dialects/dialect.py

```diff
@@ -142,7 +142,7 @@ def generator(self, **opts):
         return self.generator_class(
             **{
                 "quote": self.tokenizer_class.QUOTES[0],
-                "identifier": self.identifier,
+                "identifier": list(self.tokenizer.identifiers.items())[0],
```

Comment:
this is kind of a nit, but i think this might hurt performance a bit, we can probably do this in the line 58 section so that we don't have to do a list of items every time generator is called